Makefile

```diff
@@ -3,29 +3,32 @@ TRAVIS_BUILD_NUMBER?=debug0
 
 .PHONY: coverage get test clean
 
-all: get coverage AWSnycast
+all: _vendor coverage AWSnycast
 
-AWSnycast: *.go */*.go
-	go build -a -tags netgo -ldflags '-w' .
-	strip AWSnycast
+_vendor: Gomfile
+	gom install
+
+_vendor/src/github.com/stretchr/testify/assert: Gomfile
+	gom -test install
 
-test:
-	go test -short ./...
+AWSnycast: *.go */*.go _vendor
+	gom build -a -tags netgo -ldflags '-w' .
+	strip AWSnycast
 
-get:
-	CGO_ENABLED=0 go get -a -x -installsuffix cgo -ldflags '-d -s -w' && go install -a -x -installsuffix cgo -ldflags '-d -s -w'
+test: _vendor/src/github.com/stretchr/testify/assert
+	gom test -short ./...
```

Comment:
This seems to be running tests from the vendored packages :/ afaict from the travis runs for go1.6 .. 
do we want to do the PKGS part I added in the the Makefile from my branch and run 
https://github.com/bobtfish/AWSnycast/pull/15/files#diff-b67911656ef5d18c4ae36cb6741b7965R12
`$(foreach pkg, $(PKGS), gom test -short $(pkg);)` ?